src/github/push.ts

```diff
@@ -51,7 +51,8 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient, _u
 		const modifiedFiles = context.payload?.commits?.reduce((acc, commit) =>
 			([...acc, ...commit.added, ...commit.modified, ...commit.removed]), []);
 		// TODO: this call must be updated to support GitHub Server events
-		await updateRepoConfig(subscription, payload.repository.id, getInstallationId(gitHubInstallationId), modifiedFiles);
+		const client = await createInstallationClient(gitHubInstallationId, jiraHost, { trigger: "webhook", subTrigger: "push" }, context.log, gitHubAppId);
```

Comment:
same as above with more descriptive name

Reply:
fixed, as `gitHubInstallationClient`, to keep it consistent as the Class name and other files.